### PR TITLE
fix: Flagsmith provider no key exception

### PR DESCRIPTION
**Pull request by vpetrusevici**
## This PR

- fixed possible exception when targetKey was not set in context.

### CODE REVIEW

**Review by matthewelwell — APPROVED** [review]

**Comment by vpetrusevici** [comment]
[toddbaert](https://github.com/toddbaert) Hi
Can we complete this one&

File: src/OpenFeature.Contrib.Providers.Flagsmith/OpenFeature.Contrib.Providers.Flagsmith.csproj [hunk 1]
```diff
@@ -3,7 +3,7 @@
   <PropertyGroup>
     <TargetFrameworks>netstandard20</TargetFrameworks>
     <PackageId>OpenFeature.Contrib.Providers.Flagsmith</PackageId>
-    <VersionNumber>0.1.0</VersionNumber>
+    <VersionNumber>0.1.1</VersionNumber>
```
Comment (beeme1mr) [inline src/OpenFeature.Contrib.Providers.Flagsmith/OpenFeature.Contrib.Providers.Flagsmith.csproj:6]: This doesn't need to be manually adjusted.

**Review by beeme1mr — CHANGES_REQUESTED** [review]
The change looks good but please revert the version change because the release automation system will bump it automatically.

File: src/OpenFeature.Contrib.Providers.Flagsmith/OpenFeature.Contrib.Providers.Flagsmith.csproj (hunk 1 above)
Comment (vpetrusevici) [reply src/OpenFeature.Contrib.Providers.Flagsmith/OpenFeature.Contrib.Providers.Flagsmith.csproj]: done

**Pull request merged**